### ISSUE TITLE
How to update PhantomJS?

### ISSUE THREAD

**Issue by Luc45** [microweber/screen#99]
Hello,

How do I update PhantomJS to the latest version?

Thanks

**Comment by peter-mw** [comment]
you will need to replace the file in the /bin/ folder 

just to know that the PhantomJS project is deprecated

**Comment by Luc45** [comment]
Yeah, I wanted to update it because the screenshot generated wasn't exactly the same as the live website.

Do you know any alternative to take screenshots, preferably using PHP and avoiding third-party APIs?
